docs/en/observability/monitor-infra/host-details-partial.asciidoc

```diff
@@ -52,18 +52,39 @@ along with the total number of processes in these various states:
 The processes listed in the *Top processes* table are based on an aggregation of the top CPU and the top memory consuming processes.
 The number of top processes is controlled by `process.include_top_n.by_cpu` and `process.include_top_n.by_memory`.
 
-|=== 
+|===
 
 | *Command* | Full command line that started the process, including the absolute path to the executable, and all the arguments (`system.process.cmdline`).
 | *PID* | Process id (`process.pid`).
 | *User* | User name (`user.name`).
 | *CPU* | The percentage of CPU time spent by the process since the last event (`system.process.cpu.total.pct`).
-| *Time* | The time the process started (`system.process.cpu.start_time`). 
-| *Memory* | The percentage of memory (`system.process.memory.rss.pct`) the process occupied in main memory (RAM). 
+| *Time* | The time the process started (`system.process.cpu.start_time`).
+| *Memory* | The percentage of memory (`system.process.memory.rss.pct`) the process occupied in main memory (RAM).
 | *State* | The current state of the process and the total number of processes (`system.process.state`). Expected values are: `running`, `sleeping`, `dead`, `stopped`,
 `idle`, `zombie`, and `unknown`.
 
-|=== 
+|===
+====
+
+[%collapsible]
+.*Universal Profiling*
+====
+
+[role="screenshot"]
+image::images/universal-profiling-overlay.png[Host Universal Profiling]
+
+The *Universal Profiling* tab shows CPU usage down to the application code level.
+From here, you can find the sources of resource usage, and identify code that can be optimized to reduce infrastructure costs.
+The Universal Profiling tab has the following views.
```

Comment:
Small suggestion, we might want to also point people to [the full Profiling documentation](https://www.elastic.co/guide/en/observability/current/universal-profiling.html).